src/Redux/api.tsx

```diff
@@ -354,7 +355,7 @@ const routes = {
 
   getFacilityUsers: {
     path: "/api/v1/facility/{facility_id}/get_users/",
-    TRes: Type<PaginatedResponse<UserModel>>(),
+    TRes: Type<PaginatedResponse<UserAssignedModel>>(),
```

Comment:
Why was this changed?

`UserAssignedModel` contains less information than what the response gives and is not the right type. Revert it to `UserModel`.
```suggestion
    TRes: Type<PaginatedResponse<UserModel>>(),
```

Reply:
@rithviknishad UserModel doesn't have the skills attribute. When we are fetching the facility users here, we are also receiving the user's skills.
![image](https://github.com/coronasafe/care_fe/assets/63492939/b50c9b90-1981-461d-860e-8659a18c1c9d)